old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/04-intro-CRS-projection.Rmd

```diff
@@ -463,17 +463,18 @@ the graphic below optimize?
 ## north...
 
 # What each CRS optimizes:
-## Mercator:
-## ALbers Equal Area
-## UTM Zone 11n
-## Geographic WGS84 (lat/lon):
+## Mercator: distance
+## Albers Equal Area: area
+## UTM Zone 11n: distance
+## Geographic WGS84 (lat/lon): shape
```

Comment:
@donal-at-NEON: I'm not sure if it was intentional to leave out the answers, but I added them. I'm not confident that they're the correct answers though! Please double-check and make sure I'm not leading our tutorial users astray.

Reply:
This chunk had the `echo=FALSE` code at the top already. Let me know if `results="hide` is needed as well. I changed the Mercator answer to "shape" because the question asked if shape, area, or distance is best represented by each of those projections.

Reply:
Sounds good! You are correct that 'shape' is the better answer as it is listed above.